legacy_packages/react/src/wallet/wallets/imtoken/imTokenWallet.tsx

```diff
@@ -24,9 +24,13 @@ export const imTokenWallet = (
     id: ImTokenWallet.id,
     recommended: options?.recommended,
     meta: {
-      ...ImTokenWallet.meta,
```

Comment:
This should not be changed, the updated metadata will be inherited just fine here. (You updated it on the wallet class already.)

Reply:
fix it